src/guide/advanced/triggers.md

```diff
@@ -248,9 +258,9 @@ Only the first type of event filter is implemented now, and the other two can be
 
 ### Why not WASM
 
-The above observation can be generalised. WASM can do any logic that a Turing complete machine could using the data available via queries. So in theory for event-based triggers, you could create an `AcceptAll` event filter and do all of the processing using the key-value store as persistent storage, and determining if you want to execute using easy to understand Rust code, and not our admittedly cumbersome, `EventFilters`. We don't want that. WASM takes up significantly more space, and takes longer to execute compared to plain ISI, which are slower than `EventFilters`. We want you to want to use the `EventFilters` because they would make the process much more efficient, and are working tirelessly to make the experience of using event filters much less cumbersome. 
+The above observation can be generalised. WASM can do any logic that a Turing complete machine could using the data available via queries. So in theory for event-based triggers, you could create an `AcceptAll` event filter and do all of the processing using the key-value store as persistent storage, and determining if you want to execute using easy to understand Rust code, and not our admittedly cumbersome, `EventFilters`. We don't want that. WASM takes up significantly more space, and takes longer to execute compared to plain ISI, which are slower than `EventFilters`. We want you to want to use the `EventFilters` because they would make the process much more efficient, and are working tirelessly to make the experience of using event filters much less cumbersome.
```

Comment:
```suggestion
The above observation can be generalised. WASM can do any logic that a Turing complete machine could, using the data available via queries. So in theory for event-based triggers, you could create an `AcceptAll` event filter and do all of the processing using the key-value store as persistent storage, then determining if you want to execute using easy-to-understand Rust code, and not our admittedly cumbersome, `EventFilters`. We don't want that. WASM takes up significantly more space and takes longer to execute compared to plain ISI, which are slower than `EventFilters`. We want you to want to use the `EventFilters` because they would make the process much more efficient, and we are working tirelessly to make the experience of using event filters much less cumbersome.
```